Makefile

```diff
@@ -9,15 +18,15 @@ run: _data/projects.json
 		$(JEKYLL_DOCKER_IMAGE) -- \
 		jekyll serve --livereload --livereload-port 4001
 
-# Build (output is in _site)
+build:	## build output is in _site
```

Comment:
One last thing I just noticed, please either use tabs or spaces for the comments after the target name.
E.g., here a tab is used, but for the help and run target spaces are used to separate the target name from the comment `##`.
After that I'll go ahead and merge it.

Reply:
Sorry, a problem of consistence. Done